tlair/src/main/scala/at/forsyte/apalache/tla/lir/io/Printer.scala

```diff
@@ -302,3 +302,148 @@ object SimplePrinter extends Printer {
 object FullPrinter extends Printer {
   override def apply(p_ex: TlaEx): String = ""
 }
+
+object UsableAsIdentifierPrinter extends Printer {
```

Comment:
I think it would be good to include a Scaladoc comment with an example.
It's a bit difficult to see from the code how this prints.